### PR TITLE
feat: added splitHttp nginx reverse proxy exmaple

### PR DESCRIPTION
**Pull request by iamtrazy**
This is my sample configuration for new splitHttp transport introduced in xray core v1.8.16. with nginx as a reverse proxy

### CODE REVIEW

**Comment by yuhan6665** [comment]
Looks great! Thank you

**Comment by Fangliding** [comment]
I noticed that there seems to be a regex in nginx config for matching UUID, it can be removed

**Comment by yuhan6665** [comment]
Looks like I merge too quick. Please keep discussing..

File: VLESS-TLS-splitHttp-Nginx/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+            proxy_set_header Host $proxy_host;
+
+            proxy_http_version 1.1;
+            proxy_cache_bypass $http_upgrade;
+
+            proxy_ssl_server_name off;
+
+            proxy_set_header Upgrade $http_upgrade;
+            proxy_set_header Connection $connection_upgrade;
```
[the remark this answers is not in the document]
Comment (iamtrazy) [reply VLESS-TLS-splitHttp-Nginx/nginx.conf:61]: okay i will reduce it to the basic idea and ad a new pull request just as in vless-wss-nginx

**Pull request merged**